src/query/storage/cache/redis_cache_test.go

```diff
@@ -0,0 +1,234 @@
+// Simple unit tests for Redis cache
+// This test suite requires setting up a Redis instance at
+// 127.0.0.1:6379 (localhost)
+
+package cache
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/m3db/m3/src/query/block"
+	"github.com/m3db/m3/src/query/generated/proto/prompb"
+	"github.com/m3db/m3/src/query/models"
+	"github.com/m3db/m3/src/query/storage"
+	radix "github.com/mediocregopher/radix/v3"
```

Comment:
Is there a particular reason we use a personal developers' redis golang reop instead of the offical one: https://github.com/go-redis/redis

Reply:
Yeah I was basing it off of the ratelimit code which also uses this package, but switching to go-redis shouldn't be that hard of a change